### PR TITLE
Clarifying WebID requirement

### LINKED ISSUE #166: Requiring WebIDs?
The [Solid spec](https://github.com/solid/web-access-control-spec#overview) requires to use of WebIDs.
[We currently support strings and URIs for acl:Agent objects](https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate#WebACAuthorizationDelegate-Authorizations).

Do we want to change this to WebIDs?

### LINKED ISSUE #200: Group URIs must be dereferencable
Related to #166, the editors decided that group URIs must be dereferencable, and must be RDF, and must link to member URIs with `vcard:hasMember`.  These are the minimum requirements for clients and servers to be able to figure out which users are included in a group.

### PR DESCRIPTION
**Pull request by escowles**
Fixes #200, Fixes #166, Supersedes #199

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -888,6 +888,26 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
...
+        <blockquote class="informative">
+          <p>
+            Non-normative note:
+            The [[!SOLIDWEBAC]] <a href="https://github.com/solid/web-access-control-spec#overview">overview</a>
+            mentions parenthetically that user and group identifiers should be WebIDs.  For the purposes of performing
+            authorization, the essential requirements are:
+          </p>
+          <ul>
+            <li>
+              User and group identifiers must be IRIs.
```
Comment (acoburn) [inline index.html]: One key piece about WebIDs is that they are not just IRIs but also URLs. Is that requirement not part of this? Requiring a URL is implied below in that "Group IRIs must be dereferenceable", so would it make sense to state that more explicitly? And I assume from this that User IRIs don't necessarily need to be URLs, correct?

File: index.html [hunk 2]
```diff
@@ -888,6 +888,26 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
...
+            </li>
+            <li>
+              Group IRIs must be dereferencable as RDF, and must link to member IRIs using the
```
Comment (acoburn) [inline index.html]: dereferencable => dereferenceable

Comment (acoburn) [reply index.html]: also, the comma is not necessary.

**Review by zimeon — APPROVED** [review]

**Review by awoods — APPROVED** [review]

**Review by barmintor — APPROVED** [review]
Thanks for this, escowles.

**Review by dannylamb — APPROVED** [review]

**Pull request merged**